removes unused 'merge' linear layers in ptr gen

Adamits opened the pull request.

kylebgorman reviewed (approved):
LGTM, surprised it's that easy. I'll test locally before merging.

Adamits commented:
I ran a quick test on my mac with an ptr-gen LSTM. Did not wait for it to converge though. I think we replaced the initialization a while back and just missed these when we updated, so its just basic cleanup unless I missed something.
